contrib_bots/test-bots

```diff
@@ -17,11 +18,11 @@ def dir_join(dir1, dir2):
 if __name__ == '__main__':
     description = 'Script to run test_<bot>.py files in bots/<bot> directories'
     parser = argparse.ArgumentParser(description=description)
-    parser.add_argument('--bot',
+    parser.add_argument('--only',
                         nargs=1,
                         type=str,
                         action='store',
-                        help='test specified single bot')
+                        help='specified single bot (or comma-separated list of bots)')
```

Comment:
the convention we have in `test-backend`is that if you pass any positional arguments, those are the names of tests to run -- no need for a `--foo` argument at all.  Maybe we should go with that approach here as well?